src/pytest_html/scripts/datamanager.js

```diff
@@ -47,6 +53,10 @@ class DataManager {
     get environment() {
         return this.renderData.environment
     }
+
+    get initialSort() {
+        return this.data.initialSort
```

Comment:
`return [...this.data.initialSort]`

This prevents data.initialSort from becoming compromised by being updated by reference.